### ISSUE TITLE
Error upon loading any model

### ISSUE THREAD

**Issue by sqwyd** [cloveranon/Clover-Edition#171]
After the recent commits, I have found myself unable to load any model in the program, even though they were working fine just before.

This is a list of my current models folder, to show it matches the specifications.

```models
|-- EleutherAI-gpt-neo
|   |-- config.json
|   |-- merges.txt
|   |-- pytorch_model.bin
|   `-- vocab.json
|-- EleutherAI-h-gpt-neo
|   |-- config.json
|   |-- merges.txt
|   |-- pytorch_model.bin
|   `-- vocab.json
|-- EleutherAI-ln-gpt-neo
|   |-- config.json
|   |-- merges.txt
|   |-- pytorch_model.bin
|   `-- vocab.json
`-- pytorch_model.bin
```
The pytorch_model.bins in the directories are all hard links to the model in the models directory itself, however I have also had this issue earlier when trying to load a GPT2 model, so it's likely not a linking issue, and this also rules out it being a GPT2 or GPT-Neo issue.

Attached are hopefully useful logs of running the program with Python's verbose mode (./venv/bin/python -v launch.py), and running the program normally (./venv/bin/python launch.py), both with log-level set to 0. Both are also using the python in the venv/ folder, instead of the system's native python. I have also attached my config.ini, in case it's important. I

[regular.txt](https://github.com/cloveranon/Clover-Edition/files/6477347/regular.txt)

[verbose.txt](https://github.com/cloveranon/Clover-Edition/files/6477348/verbose.txt)

(and I've messed up and named them the opposite of what they should be... apologies!)

[config.ini.txt](https://github.com/cloveranon/Clover-Edition/files/6477367/config.ini.txt)

If it helps, I'm using Linux.

**Comment by MikkoMMM** [comment]
I got this personally, after I made it so the try-except didn't hide the errors:
```
Enter a number from above (default 0):1
Traceback (most recent call last):
  File "/mnt/lvdata/games/AIDungeon2/Clover-Edition/aidungeon/__init__.py", line 46, in <module>
    gm = GameManager(get_generator())
  File "/mnt/lvdata/games/AIDungeon2/Clover-Edition/aidungeon/play.py", line 45, in get_generator
    generator = GPT2Generator(
  File "/mnt/lvdata/games/AIDungeon2/Clover-Edition/aidungeon/gpt2generator.py", line 250, in __init__
    with open(str(model_path) + "\\config.json") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'models/gpt2\\config.json'

A fatal error has occurred. 
Traceback (most recent call last):
  File "/mnt/lvdata/games/AIDungeon2/Clover-Edition/aidungeon/__init__.py", line 46, in <module>
    gm = GameManager(get_generator())
  File "/mnt/lvdata/games/AIDungeon2/Clover-Edition/aidungeon/play.py", line 45, in get_generator
    generator = GPT2Generator(
  File "/mnt/lvdata/games/AIDungeon2/Clover-Edition/aidungeon/gpt2generator.py", line 250, in __init__
    with open(str(model_path) + "\\config.json") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'models/gpt2\\config.json'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/lvdata/games/AIDungeon2/Clover-Edition/launch.py", line 1, in <module>
    import aidungeon
  File "/mnt/lvdata/games/AIDungeon2/Clover-Edition/aidungeon/__init__.py", line 61, in <module>
    if gm and gm.story:
NameError: name 'gm' is not defined
code 1
```

**Comment by sqwyd** [comment]
Changing the "`//config.json`" on line 254 to "`/config.json`" seems to have fixed this error on Linux!

Would changing it from `\\` to `/` cause compatibility errors for Windows users?

**Comment by MikkoMMM** [comment]
I think we should use `os.path.join`.

Because this was a direct push and not a pull request, this managed to happen in the first place. But I suppose it's not easy to predict what can potentially break stuff and what can't.

**Comment by MikkoMMM** [comment]
Closed by #172.
